tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -143,6 +143,7 @@ public function userCreatesAPublicLinkShareWithSettings(string $user, TableNode
 
 		$bodyRows['displayName'] = \array_key_exists('displayName', $bodyRows) ? $bodyRows['displayName'] : null;
 		$bodyRows['expirationDateTime'] = \array_key_exists('expirationDateTime', $bodyRows) ? $bodyRows['expirationDateTime'] : null;
+		$bodyRows['password'] = \array_key_exists('password', $bodyRows) ? $bodyRows['password'] : null;
```

Comment:
```suggestion
		$bodyRows['password'] = $bodyRows['password'] ?? null;
```
I think, simply this can be done.